packages/table/examples/table/pages/demo/demo.page.ts

```diff
@@ -0,0 +1,149 @@
+import { Component } from '@angular/core';
+import { DatePipe } from '@angular/common';
+
+import { TableColumn } from '@acpaas-ui/ngx-components/table';
+import { TableActionComponent } from '../../components/table-action.component';
+
+@Component({
+	templateUrl: './demo.page.html',
+	providers: [
+		DatePipe,
+	],
+})
+export class TableDemoPageComponent {
+	public columns: TableColumn[] = [
+		{
+			label: '#',
+			value: 'id',
+		},
+		{
+			label: 'First Name',
+			value: 'firstName',
+		},
+		{
+			label: 'Last Name',
+			value: 'lastName',
+		},
+		{
+			label: 'Registered',
+			value: 'registeredAt',
+			format: (value) => this.datePipe.transform(value, 'dd/MM/yyyy'),
+		},
+		{
+			label: 'Actions',
+			component: TableActionComponent,
+		},
+	];
+	/* tslint:disable */
+	public rows = [
+		{
+			"id": 0,
```

Comment:
where did you get the names?

Reply:
just some randomly generated names: https://next.json-generator.com/